pkgs/servers/jitsi-videobridge/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchurl, dpkg, jre_headless, nixosTests }:
+
+let
+  pname = "jitsi-videobridge2";
+  version = "2.1-164-gfdce823f";
+  src = fetchurl {
+    url = "https://download.jitsi.org/stable/${pname}_${version}-1_all.deb";
+    sha256 = "0n4c5iz5gwsa142ihm1bd48k71crwls6fgfm4i0wgzvvxhgs213a";
+  };
+in
+stdenv.mkDerivation {
+  inherit pname version src;
+
+  phases = [ "unpackPhase" "installPhase" "fixupPhase" ];
+
+  unpackPhase = ''
```

Comment:
Please set `unpackCmd = "${dpkg}/bin/dpkg-deb -x $src .";` instead (and remove `dpkg` from `nativeBuildInputs)